APCAppCore/APCAppCore/Library/DataArchiverAndCollector/Trackers/APCCoreLocationTracker.m

```diff
@@ -126,6 +127,7 @@ - (void)startTracking
     if ([CLLocationManager locationServicesEnabled] == YES)
```

Comment:
- (void)startTracking
 {
     if ([CLLocationManager locationServicesEnabled] == YES)
     {
         APCLogDebug(@"Start location tracking");
+        self.locationManager.delegate = nil;
         self.locationManager = [[CLLocationManager alloc] init];
         self.locationManager.delegate = self;

just curious why you're allocating a whole new CLLocationManager every time you start and stop tracking, rather than creating it once and then starting/stopping SignificantLocationChanges as needed...?